--- youtube-dl	2019-05-08 06:24:14.958885288 +0700
+++ youtube-dl-new	2019-05-08 06:22:25.895946184 +0700
@@ -10,3 +10,17 @@
     sys.exit(
         load_entry_point('youtube-dl==2019.1.17', 'console_scripts', 'youtube-dl')()
     )
+
+diff --git a/youtube_dl/InfoExtractors.py b/youtube_dl/InfoExtractors.py
+index baf859e..01ec034 100644
+--- a/youtube_dl/InfoExtractors.py
++++ b/youtube_dl/InfoExtractors.py
+@@ -2357,7 +2357,7 @@ class XVideosIE(InfoExtractor):
+
+
+        # Extract video thumbnail
+-       mobj = re.search(r'http://(?:img.*?\.)xvideos.com/videos/thumbs/[a-fA-F0-9]/[a-fA-F0-9]/[a-fA-F0-9]/([a-fA-F0-9.]+jpg)', webpage)
++       mobj = re.search(r'http://(?:img.*?\.)xvideos.com/videos/thumbs/[a-fA-F0-9]+/[a-fA-F0-9]+/[a-fA-F0-9]+/[a-fA-F0-9]+/([a-fA-F0-9.]+jpg)', webpage)
+        if mobj is None:
+            self._downloader.trouble(u'ERROR: unable to extract video thumbnail')
+            return
